src/pages/lists/status-list/index.tsx

```diff
@@ -89,23 +247,177 @@ export const StatusList = (): JSX.Element => {
-                    <div />
+                    <Spacer />
+                    <IconButton color={'inherit'} edge={'end'}>
+                        <HelpIcon />
+                        <Badge classes={{ root: classes.headerIcon }} color="error" badgeContent={88}>
+                            <NotificationIcon />
+                        </Badge>
+                        <MoreVertIcon classes={{ root: classes.headerIcon }} />
+                    </IconButton>
                 </Toolbar>
             </AppBar>
-            <List className={'list'} disablePadding>
-                {list.map((item, i) => (
-                    <InfoListItem
-                        iconColor={theme.palette.text.primary}
-                        statusColor={'transparent'}
-                        key={i}
-                        avatar
-                        divider={'partial'}
-                        {...item}
-                    />
-                ))}
-            </List>
+            <div className={classes.accordionContainer}>
+                <Accordion
+                    key={'With Time Stamps, with Title+SubTitle+Info'}
+                    data-testid="statusListAccordion"
+                    defaultExpanded={true}
+                    classes={{ root: classes.accordionRoot }}
+                >
+                    <AccordionSummary expandIcon={<ExpandMore />}>
+                        <Typography variant={'subtitle2'} color={'primary'}>
+                            With Time Stamps, with Title+SubTitle+Info
+                        </Typography>
+                    </AccordionSummary>
+                    <AccordionDetails classes={{ root: classes.accordionDetails }}>
+                        <List className={'list'} disablePadding>
+                            <InfoListItem
+                                classes={{
+                                    listItemText: classes.listItemText,
+                                }}
+                                title={getTitle('High Humidity', 'PX341 sensor level 9', isMobile, classes)}
+                                data-testid="statusListInfoListItem"
+                                subtitle={getSubtitle('Cherrington Station', 'Moon Township', classes)}
+                                subtitleSeparator={' '}
+                                info={getInfoComponent(isMobile, classes, true)}
+                                icon={<NotificationsActiveIcon />}
+                                iconColor={theme.palette.common.white[50]}
+                                statusColor={colors.red[500]}
+                                leftComponent={getLeftComponent('8:21', 'AM', '11/23/21', classes)}
+                                rightComponent={getRightComponent(isMobile, classes, true)}
+                                divider={'partial'}
+                                avatar
+                                chevron
+                            />
+                            <InfoListItem
```

Comment:
You would want the `iconAlign` prop of all `<InfoListItem>` set to `'center'`. Read more about the rule here: https://pxblue.github.io/patterns/lists#lists-with-status

Reply:
Done.